### PR TITLE
docs: update readme, add custom metrics, better formating

### PR DESCRIPTION
**Pull request by abtris**
- add section about custom metrics
- update formatting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,3 +64,22 @@ ALLOWED_APPS=my-app,..    # Required. Comma seperated list of app names
...
+
+## Heroku settings
+
+You need use Standard dynos and better and enable `log-runtime-metrics` in heroku labs for every application.
+
+```bash
+heroku labs:enable log-runtime-metrics -a APP_NAME
+```
+
+This add into logs are metrics what you need (cpu, memory etc.).
```
Comment (w-vi) [inline README.md]: This adds basic metrics (cpu, memory etc.) into logs.

File: README.md [hunk 2]
```diff
@@ -47,3 +64,22 @@ ALLOWED_APPS=my-app,..    # Required. Comma seperated list of app names
...
+
+## Custom Metrics
+
+If you can log some your custom metrics just log line in this format:
```
Comment (w-vi) [inline README.md]: If you want to log some custom metrics just format the log line like following:

File: README.md [hunk 3]
```diff
@@ -47,3 +64,22 @@ ALLOWED_APPS=my-app,..    # Required. Comma seperated list of app names
...
+
+```
+app web.1 - info: responseLogger: metric#tag#route=/parser metric#request_id=11747467-f4ce-4b06-8c99-92be968a02e3 metric#request_length=541 metric#response_length=5163 metric#parser_time=5ms metric#eventLoop.count=606 metric#eventLoop.avg_ms=515.503300330033 metric#eventLoop.p50_ms=0.8805309734513275 metric#eventLoop.p95_ms=3457.206896551724 metric#eventLoop.p99_ms=3457.206896551724 metric#eventLoop.max_ms=5008
+```
+We support `metrics#` for values and `metrics#tag` for tags.
```
Comment (w-vi) [inline README.md]: Shouldn't it be `metric#` for values and `metric#tag#`?

**Review by w-vi — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (abtris) [reply README.md]: Good catch!

**Comment by abtris** [comment]
Updated.

**Review by w-vi — APPROVED** [review]

**Pull request merged**